src/rdkafka.h

```diff
@@ -3368,14 +3456,19 @@ void rd_kafka_log_syslog(const rd_kafka_t *rk, int level,
 /**
  * @brief Returns the current out queue length.
  *
- * The out queue contains messages waiting to be sent to, or acknowledged by,
- * the broker.
+ * The out queue length is the sum of:
+ *  - number of messages waiting to be sent to, or acknowledged by,
+ *    the broker.
+ *  - number of delivery reports waiting
```

Comment:
ahh, i see this is better documented now. I think you could still add a little more verbosity to this point and the next.